release/handbrake.xml

```diff
@@ -22,26 +18,66 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app HandBrake
+    @url https://handbrake.fr/
+    @os Windows, Linux
+    @cleanerversion v0.5.0
+    @cleanerdate 2019-03-18
+    @cleanerby Rogério Theodoro de Brito (2012 - 2013) & Erratalog (2016) & Tobias B. Besemer (2019-03-18)
+    @tested ok v???, Windows ???
+    @testeddate ???
+    @testedby ???
+    @note This cleaner makes the WinApp2.ini entries to Handbrake necessary!
+
 -->
 <cleaner id="handbrake">
   <label>HandBrake</label>
   <description>Video transcoder</description>
+  <!-- Not yet figured out:
+  <running type="exe" os="linux">handbrake</running>
+  -->
+  <running type="exe" os="windows">Handbrake</running>
+  <var name="ProgramFiles">
+    <!--
+    Linux $$ProgramFiles$$ not yet figured out! I will do this later! Tobias.
+    <value os="linux">~/.config/ghb</value>
+    -->
+    <!--
+    Windows itself changes %ProgramFiles% for 32-bit processes running in a 64-bit operating system as documented here:
+    https://docs.microsoft.com/en-us/windows/desktop/winprog64/wow64-implementation-details
+    ...so we use %SystemDrive%\Program Files\ & %SystemDrive%\Program Files (x86)\...
+    -->
+    <value os="windows">%SystemDrive%\Program Files\HandBrake</value>
+    <value os="windows">%SystemDrive%\Program Files (x86)\HandBrake</value>
```

Comment:
According to this [Wikipedia article on Program Files](https://en.wikipedia.org/wiki/Program_Files), the directory name can be localized, so these paths may break on some systems. It's best to use Program Files environment variables instead.

Can Handbreak be installed in both the `x86` and "regular"?

Reply:
> 32bit builds are no longer provided.

But were in the past...

Reply:
Would you please change the ProgramFiles here to be like BiglyBT?

Reply:
Done it already! Just didn't made a PR by now...